pytensor/tensor/basic.py

```diff
@@ -4355,28 +4355,22 @@ def empty_like(
 
 
 def atleast_Nd(
-    *arys: np.ndarray | TensorVariable, n: int = 1, left: bool = True
+    arry: np.ndarray | TensorVariable, n: int = 1, left: bool = True
```

Comment:
Just to avoid surprises, I would make it so you have to specify n explicitly like before. It simply doesn't accept multiple arrays anymore, so it won't be a silent change if someone was relying on this.

Later on we can allow n to be positional only like you did now.
```suggestion
    arry: np.ndarray | TensorVariable, *, n: int = 1, left: bool = True
```

Reply:
I have made this change, modified the tests accordingly too